grpc/src/main/proto/grpc/event_service.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+package conductor.grpc;
+
+import "google/protobuf/empty.proto";
+import "model/eventhandler.proto";
+
+option java_package = "com.netflix.conductor.grpc";
+option java_outer_classname = "EventServicePb";
+option go_package = "github.com/netflix/conductor/client/gogrpc/conductor/grpc";
+
+service EventService {
+    // POST /
+    rpc AddEventHandler(conductor.proto.EventHandler) returns (google.protobuf.Empty);
+
+    // PUT /
+    rpc UpdateEventHandler(conductor.proto.EventHandler) returns (google.protobuf.Empty);
+
+    // DELETE /{name}
+    rpc RemoveEventHandler(RemoveEventHandlerRequest) returns (google.protobuf.Empty);
+
+    // GET /
+    rpc GetEventHandlers(google.protobuf.Empty) returns (stream conductor.proto.EventHandler);
```

Comment:
e.g. create a empty GetEventHandlersRequest, and GetEventHandlersResponse with EventHandler field.